src/test/scala/edu/ie3/util/quantities/QuantityUtilSpec.scala

```diff
@@ -186,4 +187,34 @@ class QuantityUtilSpec extends UnitSpec with TableDrivenPropertyChecks {
       }
     }
   }
+
+  "Converting units to alternative units" should {
+    "succeed if units are compatible" in {
```

Comment:
Maybe add at least one test case where units are not compatible?

Reply:
I don't think that's possible, since the unit type parameter (`Q`) prevents incompatible types to be applied as parameters